.storybook/main.js

```diff
@@ -1,19 +1,11 @@
 const path = require('path');
 
 module.exports = {
-  stories: [
-    // Make Introduction appear first in list of stories.
-    '../stories/Intro.stories.mdx', 
-    // Load stories in JS format
-    // Load stories in MDX format (@storybook/addon-docs)
-    '../stories/**/*.stories.(js|mdx)', 
-    // include stories from virtru-react-components
-    '../stories/vrc-stories/*.(js|mdx)',
-  ],
+  stories: ['../stories/**/*.stories.(js|mdx)', '../stories/vrc-stories/**/*.js', '../lib/components/**/*.stories.(js|mdx)'],
```

Comment:
please keep the comments; again I don't know what's stripping this out but I'd rather keep the context